src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -227,9 +227,14 @@
    * </pre>
    *
    * by replacing the second Error with GlobalError.
+   *
+   * <p>Closure has internal "externs" for some of these symbols:
+   * https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/rhino/jstype/JSTypeRegistry.java
+   * Adding a symbol to this list has the side effect of suppressing an unnecessary declaration of
+   * the global type in each file that references it in partial mode.
```

Comment:
Hmm, I couldn't quite follow that sentence. In my mind, 'adding a symbol to this list also allows for accessing global symbols that have been shadowed'.

Even better if you add an example like the Error above.